ftd/image.ftd

```diff
@@ -259,3 +259,41 @@ dark: https://fastn.com/-/fastn.com/images/cs/show-cs-1-dark.jpg
 
 
 -- ftd.image-src just-light: https://fastn.com/-/fastn.com/images/cs/show-cs-1.jpg
+
+-- ds.h2: `fetch priority`
+
+Type: `optional` `string`
```

Comment:
Have we not added a new type `ftd.fetch-priority` or something? `string` is not allowed. You have to document all possible/legal values. Also there should be no `auto`, at least in example show something other than auto, everything is auto, what is the point of giving and example with default?